exercises/02-module-class/color-generator-2.0.mjs

```diff
@@ -0,0 +1,25 @@
+class ColorsGenerator {
+
+    colors = ['red', 'green', 'blue'];
```

Comment:
This field should be [private](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields)

From [exercise description](https://github.com/hubertkozik/highcharts-modules/tree/main/exercises/02-module-class):
> Implement a [Class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) with:
> 
> a) Public methods:
> 
> `get(n?: number): Array<string>;`
> `reset(): void;`
**> b) Private properties and methods:**
> 
> `colors: Array<string>`
> any other properties and methods you need